CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+# 2.0.6
```

Comment:
```suggestion
# 2.1.0
```
I think bumping the minor version instead of the patch would be more appropriate for dependencies update.